src/background/messages/fetchProjectDigest.ts

```diff
@@ -0,0 +1,42 @@
+import type { PlasmoMessaging } from "@plasmohq/messaging"
+
+import { eventDigestQueryString } from "~queries/eventDigestQuery"
+import { FullProject } from "~types/projectTypes"
+import { UserSession } from "~types/userTypes"
+import determineActiveProjectId from "~utils/determineActiveProjectId"
+import { fetchStrapiContent } from "~utils/fetchStrapiContent"
+import generatePagination from "~utils/generatePagination"
+import newStorage from "~utils/newStorage"
+
+const handler: PlasmoMessaging.MessageHandler = async (req, res) => {
+  const storage = newStorage()
+  const userSession: UserSession = await storage.get(
+    "arebyte-audience-session"
```

Comment:
Now with all the fighting we've been doing with storage I wonder if we want to store this in the user content type and then just fetch it in our initial user fetch and store it in the user store context object.

Reply:
cool I've now refactored the feature to reference a new field in the cms after merging the PR with the new data flow 👍